dev/custom-pages/homepage.scss

```diff
@@ -270,8 +269,29 @@ $color-linkedin: #0077b5;
     }
 
     .padded-bottom {
-      padding-bottom: 70px;
+      padding-bottom: 47px;
     }
+
+    @media (max-width: $screen-md-max) {
```

Comment:
Changes were not unrelated and are being kept. Updated to have each rule have the media query within it.